config/http.conf

```diff
@@ -1,18 +1,33 @@
+daemon off;
+
+worker_processes auto;
+worker_shutdown_timeout 240s;
+
+events {
+  multi_accept        on;
+  worker_connections  16384;
+  use                 epoll;
+}
+
 http {
   include /etc/nginx/mime.types;
   include /etc/nginx/log.conf;
 
+  aio       threads;
+  aio_write on;
+
   server_tokens           off;
   sendfile                on;
   tcp_nopush              on;
+  tcp_nodelay             on;
 
   client_max_body_size    500m;
-  client_body_buffer_size 128k;
-  client_body_timeout     300s;
-  client_header_timeout   605s;
-  keepalive_timeout       605s;
-  proxy_connect_timeout   60s;
-  proxy_read_timeout      600s;
+  client_body_buffer_size 16k;
+  client_body_timeout     60s;
+  client_header_timeout   60s;
+  keepalive_timeout       75s;
+  proxy_connect_timeout   5s;
+  proxy_read_timeout      60s;
```

Comment:
All of these values are based on ingress-nginx's defaults.

Reply:
Good start. This was a good suggestion by the way!